contracts/transfer/tests/transfer.rs

```diff
@@ -1058,3 +1058,137 @@ fn transfer_to_contract() {
         "Bob must have the transfer value as balance"
     );
 }
+
+/// In this test we deposit some Dusk to the Alice contract, and subsequently
+/// call the Alice contract to trigerring a transfer back to the same address.
```

Comment:
/// In this test we deposit some Dusk **from a Moonlight account** to the Alice contract, and subsequently
/// call the Alice contract to trigerring a transfer back to the same ~~address~~ **account**.

Reply:
Fixed